runtime/syntax/stylus.vim

```diff
@@ -0,0 +1,380 @@
+syn match cssPseudo /::\(first-\(line\|letter\)\|before\|after\|selection\)/
+
+" CSS3 Advanced http://meiert.com/en/indices/css-properties/
+syn keyword cssAdvancedProp appearance azimuth binding bleed columns crop hyphens icon
+syn keyword cssAdvancedProp phonemes resize richness size volumne
+syn match cssAdvancedProp /\(animation-\(delay\|direction\|duration\|name\|iteration-count\|play-state\|timing-function\)\|animation\)/
+syn match cssAdvancedProp /alignment-\(adjust\|baseline\)/
+syn match cssAdvancedProp /\(backface-visibility\baseline-shift\)/
+syn match cssAdvancedProp /bookmark-\(label\|level\|state\|target\)/
+syn match cssAdvancedProp /column-\(count\|fill\|gap\|rule-\(color\|style\|width\)\|rule\|span\|width\)/
+syn match cssAdvancedProp /\(cue-\(after\|before\)\|cue\)/
+syn match cssAdvancedProp /dominant-baseline/
+syn match cssAdvancedProp /drop-initial-\(size\|value\|\(after\|before\)-\(adjust\|align\)\)/
+syn match cssAdvancedProp /\(fit-position\|fit\)/
+syn match cssAdvancedProp /\(float-offset\|hanging-punctuation\)/
+syn match cssAdvancedProp /grid-\(columns\|rows\)/
+syn match cssAdvancedProp /hyphenate-\(after\|before\|character\|lines\|resource\)/
+syn match cssAdvancedProp /image-\(orientation\|rendering\|resolution\)/
+syn match cssAdvancedProp /inline-box-align/
+syn match cssAdvancedProp /\(mark-\(after\|before\)\|mark\|marks\)/
+syn match cssAdvancedProp /marquee-\(direction\|loop\|play-count\|speed\|style\)/
+syn match cssAdvancedProp /move-to/
+syn match cssAdvancedProp /nav-\(down\|index\|left\|right\|up\)/
+syn match cssAdvancedProp /\(pause-\(after\|before\)\|pause\)/
+syn match cssAdvancedProp /\(perspective-origin\|perspective\)/
+syn match cssAdvancedProp /\(pitch-range\|pitch\)/
+syn match cssAdvancedProp /presentation-level/
+syn match cssAdvancedProp /punctuation-trim/
+syn match cssAdvancedProp /rendering-intent/
+syn match cssAdvancedProp /pointer-events/
+syn match cssAdvancedProp /\(rest-\(after\|before\)\|rest\)/
+syn match cssAdvancedProp /\(rotation-point\|rotation\)/
+syn match cssAdvancedProp /ruby-\(align\|overhang\|position\|span\)/
+syn match cssAdvancedProp /\(target-\(name\|new\|position\)\|target\)/
+syn match cssAdvancedProp /\(transform-\(origin\|style\)\|transform\)/
+syn match cssAdvancedProp /\(transition-\(delay\|duration\|property\|timing-function\)\|transition\)/
+syn match cssAdvancedProp /voice-\(balance\|duration\|family\|pitch-range\|pitch\|rate\|stress\|volume\)/
+
+syn match cssAdvancedVal /\(ease-\(in\|out\|in-out\)\|ease\)/ contained
+
+" CSS3 Advanced value
+"syn match cssAdvancedVal
+
+
+if main_syntax == "css"
+  syn sync minlines=10
+endif
+
+" Define the default highlighting.
+" For version 5.7 and earlier: only when not done already
+" For version 5.8 and later: only when an item doesn't have highlighting yet
+if version >= 508 || !exists("did_css_syn_inits")
+  if version < 508
+    let did_css_syn_inits = 1
+    command -nargs=+ HiLink hi link <args>
```

Comment:
We are not using HiLinks commands anymore

Reply:
Done.